services/proctoring-service/ai-service/venv/Lib/site-packages/joblib/test/data/create_numpy_pickle.py

```
@@ -0,0 +1,106 @@
+            kwargs["cache_size"] = 0
+            body += "_cache_size"
+    else:
+        extension = ".pkl"
+
+    pickle_filename = "joblib_{}{}_pickle_py{}_np{}{}".format(
+        joblib_version, body, py_version, numpy_version, extension
+    )
+
+    try:
+        joblib.dump(to_pickle, pickle_filename, **kwargs)
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)